zapcore/encoder.go

```diff
@@ -239,6 +239,8 @@ type EncoderConfig struct {
 	// Unlike the other primitive type encoders, EncodeName is optional. The
 	// zero value falls back to FullNameEncoder.
 	EncodeName NameEncoder `json:"nameEncoder" yaml:"nameEncoder"`
+	//More on console, the delimiter of each element
```

Comment:
- let's name it `ConsoleFoo` then rather than generically `ElementFoo`
- doc string needs to be formatted / written better, as in `// Configures the field separator used by the console encoder.` (note: leading space, final period, and phrased as a complete sentence)
- nit: in my view, this should be called a "separator" rather than a "delimiter", so `ConsoleSeparator` instead of `ElementDelimiter`
- corollary: I care less about s/delimiter/separator/ in the internal code paths above, more so in this exported struct; but probably best for to do so for internal consistency

Reply:
@jcorbin  Thanks for details instruction,  I made the changes, please have another look.